src/imitation/scripts/convert_trajs.py

```diff
@@ -1,45 +1,56 @@
-"""Converts old-style pickle trajectories to new-style NPZ trajectories.
+"""Converts old-style pickle or npz trajectories to new-style HuggingFace datasets.
 
 See https://github.com/HumanCompatibleAI/imitation/pull/448 for a description
 of the new trajectory format.
 
 This script takes as command-line input multiple paths to saved trajectories,
-in the old .pkl or new .npz format. It then saves each sequence in the new .npz
-format. The path is the same as the original but with an ".npz" extension
-(i.e. "A.pkl" -> "A.npz", "A.npz" -> "A.npz", "A" -> "A.npz", "A.foo" -> "A.foo.npz").
+in the old .pkl or .npz format. It then saves each sequence in the new HuggingFace
+datasets format. The path is the same as the original but a directory without an
+extension (i.e. "A.pkl" -> "A/", "A.npz" -> "A/", "A/" -> "A/", "A.foo" -> "A/").
 """
-
+import pathlib
 import warnings
 
+import imitation.data.serialize
```

Comment:
In other places we're importing `from imitation.data import serialize`

Reply:
Good point. Even found some more places where this is the case. I made sure it is always imported as `from imitation.data import serialize` or `from imitation import data` whenever we also need the `policies.serialize` module in the same file.